packages/react-jsx-highcharts/src/components/PlotBandLine/UsePlotBandLineLifecycle.js

```diff
@@ -25,8 +25,18 @@ export default function usePlotBandLineLifecycle(props, plotType) {
       };
 
       if (plotbandline) axis.removePlotBandOrLine(idRef.current);
+      axis.addPlotBandOrLine(opts, plotType);
       setPlotbandline({
-        object: axis.addPlotBandOrLine(opts, plotType)
+        id: myId,
+        get object() {
+          if (axis && axis.object && axis.object.plotLinesAndBands) {
+            const plotbandlineObject = axis.object.plotLinesAndBands.find(
+              plb => plb.id === myId
+            );
+            return plotbandlineObject;
+          }
+          return null;
+        }
```

Comment:
This is needed because updating parent axis recreates all plotbands and lines.

Reply:
Could we add this as a comment in the code to make that clear?